src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -8,16 +8,11 @@
             {{ "form_not_available"|trans({}, "SonataAdminBundle") }}
         </div>
     {% else %}
-        <form
-              {% if sonata_config.getOption('form_type') == 'horizontal' %}class="form-horizontal"{% endif %}
-              role="form"
-              action="{% block sonata_form_action_url %}{{ admin.generateUrl(url, {(admin.idParameter): objectId, 'uniqid': admin.uniqid, 'subclass': app.request.get('subclass')}) }}{% endblock %}"
```

Comment:
we're losing the block

Reply:
Thats right, but overriding the form action with a block looks weird. You should override the action when creating the form inside the controller.